tests/MongoDB.Driver.Core.TestHelpers/CoreTestConfiguration.cs

```diff
@@ -76,11 +77,11 @@ public static SemanticVersion ServerVersion
             get
             {
                 var server = __cluster.Value.SelectServer(WritableServerSelector.Instance, CancellationToken.None);
-                var version = server.Description.Version;
+                var description = server.Description;
+                var version = description.Version ?? (description.Type == ServerType.LoadBalanced ? _buildInfo.Value.ServerVersion : null);
```

Comment:
lb doesn't support version in a hello response, so emulating it by explicit calling of build info